src/components/FullHeader/MenuSlider.js

```diff
@@ -85,13 +85,13 @@ class MenuSlider extends React.Component {
 					>
 						{isHomepage && (
 							<Menu.Item key="1">
-								<Link to="/">Apps</Link>
+								<Link to="/">Clusters</Link>
 							</Menu.Item>
 						)}
 
 						{isHomepage && (
 							<Menu.Item key="2">
```

Comment:
Key should change here. 

Check this screenshot, we are on `/apps` page but it marks `profile` as the active page. 
![image](https://user-images.githubusercontent.com/22376783/74810982-90167c00-5316-11ea-88a1-9de7e2790cdd.png)